quarkus-workshop-super-heroes/assembly-core.xml

```diff
@@ -0,0 +1,65 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.2.0 http://maven.apache.org/xsd/assembly-2.2.0.xsd">
+    <id>workshop-core</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>true</includeBaseDirectory>
+
+    <fileSets>
+        <fileSet>
+            <outputDirectory>./</outputDirectory>
+            <directory>${basedir}/target/temp-project</directory>
+            <includes>
+                <include>mvnw*</include>
+                <include>.mvn/wrapper/maven-wrapper.properties</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <outputDirectory>./super-heroes</outputDirectory>
+            <directory>${basedir}/target/temp-project</directory>
+            <includes>
+                <include>mvnw*</include>
+                <include>.mvn/wrapper/maven-wrapper.properties</include>
+            </includes>
+        </fileSet>
+        <fileSet>
+            <directory>${basedir}</directory>
+            <useDefaultExcludes>true</useDefaultExcludes>
+            <includes>
+                <include>pom.xml</include>
+                <include>.editorconfig</include>
+                <include>mvnw*</include>
+                <include>.mvn/wrapper/maven-wrapper.properties</include>
+
+                <include>super-heroes/mvnw*</include>
+                <include>super-heroes/.mvn/wrapper/maven-wrapper.properties</include>
+
+                <include>super-heroes/infrastructure/**/*</include>
+                <include>super-heroes/kubernetes/**/*</include>
+
+                <include>super-heroes/rest-fights/*</include>
+                <include>super-heroes/rest-fights/.mvn/wrapper/maven-wrapper.properties</include>
+                <include>super-heroes/rest-fights/src/**/*</include>
+
+                <include>super-heroes/rest-heroes/*</include>
+                <include>super-heroes/rest-heroes/.mvn/wrapper/maven-wrapper.properties</include>
+                <include>super-heroes/rest-heroes/src/**/*</include>
+
+                <include>super-heroes/rest-villains/*</include>
+                <include>super-heroes/rest-villains/.mvn/wrapper/maven-wrapper.properties</include>
+                <include>super-heroes/rest-villains/src/**/*</include>
+
+                <include>super-heroes/ui-super-heroes/.mvn/wrapper/maven-wrapper.properties</include>
+                <include>super-heroes/ui-super-heroes/*</include>
+                <include>super-heroes/ui-super-heroes/src/main/docker/**/*</include>
```

Comment:
I think it would be safe to include everything in `/src`, wouldn't it?

Reply:
Well, I thought it would, but I ended up with a `node_module` directory just under `src`. I don't know if it's expected or not (there is already a `node_module` directory under `.quinoa`) but to be safe, I did it this way avoiding to include a huge `node_module` directory

Reply:
Well, with an `<exclude>` for **/node_modules/** as we had before. We can probably keep quite a few of the earlier excludes, actually, things like `<exclude>**/ObjectStore</exclude>`. Then it's kind of a belt-and-braces approach where we include generously, and filter out stuff we shouldn't have.

Reply:
Ok, let me have another look at the exclude

Reply:
Oh, very nice! That should be much more robust than what we have now.